src/Nethermind/Nethermind.JsonRpc/JsonRpcUrlCollection.cs

```diff
@@ -90,6 +90,7 @@ private void BuildEngineUrls(bool includeWebSockets)
 
     private void BuildAdditionalUrls(bool includeWebSockets)
     {
+        List<string[]> additionalRpcUrlRows = new List<string[]>();
         foreach (string additionalRpcUrl in _jsonRpcConfig.AdditionalRpcUrls)
```

Comment:
add a first row | default | <protocol list> | <modules list>|.
This contains the enabled modules / protocols for the client it self.
https://docs.nethermind.io/fundamentals/configuration/#jsonrpc-enabledmodules
https://docs.nethermind.io/fundamentals/configuration/#jsonrpc-port
https://docs.nethermind.io/fundamentals/configuration/#jsonrpc-host



the ones below are not necessary but if you could find a way to add these to the table that is clear that would be super nice

https://docs.nethermind.io/fundamentals/configuration/#jsonrpc-engineenabledmodules ( these are the protocols for the engine, similar to protocols for additionalrpc.)
https://docs.nethermind.io/fundamentals/configuration/#jsonrpc-enginehost
https://docs.nethermind.io/fundamentals/configuration/#jsonrpc-engineport